clienttools/IDEPlugins/ESDL/esdl.bat.in

```diff
@@ -32,7 +32,7 @@ echo New Input file:  %newinput% >> %TMP%\log.txt
 echo New Ouput file:  %newoutput% >> %TMP%\log.txt
 copy /Y %3 %newinput%
 
-set "clienttoolsdir=@ORIG_PREFIX@\@DIR_NAME@\@version@\clienttools"
+for /f "usebackq tokens=2*" %%a in (`reg query "HKLM\SOFTWARE" /K /F "clienttools_@version@" /S /VE ^| find "(Default)"`) do set clienttoolsdir=%%~B
```

Comment:
I think your comment in the Jira about simply using relative paths is the way to go?  
Apart from being simpler, my concern here is that you would need elevated privileges to make this query?